docs/hellonode.md

```diff
@@ -187,12 +206,14 @@ At this point you should have our container running under the control of Kuberne
 
 By default, the pod is only accessible by its internal IP within the Kubernetes cluster. In order to make the `hello-node` container accessible from outside the kubernetes virtual network, you have to expose the pod as a kubernetes **[service](/docs/user-guide/services/)**.
 
-From our development machine we can expose the pod with the `kubectl` expose command and the `--type="LoadBalancer"` flag which creates an external IP to accept traffic:
+From our development machine we can expose the pod to the public internet using the `kubectl expose` command combined with the `--type="LoadBalancer"` flag.  The flag is needed for the creation of an externally accessible ip:
 
 ```shell
 kubectl expose deployment hello-node --type="LoadBalancer"
 ```
 
+**If this fails, make sure your client and server are both version 1.2.  See the [Create your cluster](#cyc) section for details.**
```

Comment:
link to `#create-your-cluster` should already work and then we don't need to create #cyc